topics/replit/index.md

```diff
@@ -0,0 +1,9 @@
+---
+aliases: replit, repl, replit-ide
```

Comment:
Right now CI is failing:

```
topics::replit topic#test_0011_has valid aliases [/home/runner/work/explore/explore/test/topics_test.rb:133]:
alias 'replit' must not be the same as the topic.
Expected "replit" to not be equal to "replit".
```

I believe this can be fixed by removing `replit` from this list.

Reply:
OK, I'll fix that